avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,29 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, count, option=None, sudo=False):
+        """This method is used to check packet loss that occurs during
+        ping to peer address.
+
+        Function returns True for 0% packet loss and False
+        if packet loss occurs.
+
+        :param peer_ip: Peer IP address (IPv4 or IPv6)
+        :param count: How many packets to send
+        :param option: ping command options. Default is None
+        :param flood: ping flood option -f. Default is False
```

Comment:
"flood" is not used anymore in this new version. Also we should add the parameter "sudo" into the docstring.

Reply:
Removed flood from the description